doc/rst/source/cookbook/gmt-latex.rst

```diff
@@ -0,0 +1,38 @@
+Using LaTeX Expressions in GMT
+==============================
+
+GMT supports the use of LaTeX equations embedded in text strings that are used
+with **-B** in titles, subtitles and axis labels, as well as for single lines
+of text placed via :doc:`/text`.  These expressions must be enclosed by the
+marker @[ (e.g., "Plotting @[\\Delta \\sigma_{xx}^2@["), or alternatively enclose
+the expressions with <math> and </math> (e.g., "Plotting <math>\\Delta \\sigma_{xx}^2</math>").
+If these markers are found, the entire line will be converted to an
+Encapsulated PostScript file (EPS) via system commands to latex and dvips,
+which must be available on your system.  The EPS file is then placed in the
+location instead of the regular text.
+
+
+.. figure:: /_images/GMT_latex.*
+   :width: 500 px
+   :align: center
+
+   Example of using a LaTeX equation in the x-axis label via -Bxaf+l"@[\\nabla^4 \\psi - \\Delta \\sigma_{xx}^2@[ (MPa)".
+
+.. _gmt-latex-fonts:
+
+GMT fonts and LaTeX
+-------------------
+
+LaTeX is a large and complicated type-setting system with many optional packages, and we simply
+must assume that your LaTeX installation has all the needed features; see our
+`wiki <https://github.com/GenericMappingTools/gmt/wiki>`_ for typical installs
+via package managers.  Your installation must have support for the packages *fontenc*
+and *inputenc*, as well as the required fonts *helvet, mathptmx, courier, symbol,
+avantgar, bookman, newcent, mathpazo, zapfchan* and *zapfding*.  If the conversion
+from LaTeX to EPS fails, please follow instructions to run the conversion script
```

Comment:
I find the statement
>please follow instructions to run the conversion script manually' 

confusing because there is no information about where these instructions can be found. I think it would be better to say 'please run the conversion script manually' or insert a link to some instructions.

Reply:
The instructions would be printed out when things fails, including the path to the temp directory holding all the files and logs etc.  Maybe "... follow instructions in the error message to run..." ?

Reply:
It would be good if you can write some technical details about how the GMT-LaTeX integration works, i.e., tex -> dvi -> eps -> psimage.

Reply:
Yes, that change would make it clear

Reply:
OK, once I finish another branch ... I will add an example of a typical tex file and the steps.